legal_info/chamber_to_sub_area.json

```diff
@@ -0,0 +1,86 @@
+    "AR_SRK_001": "Steuerrecht",
+    "AR_RR_001": "Politische Rechte",
+    "BE_OG_007": "Schuldbetreibungs- und Konkursrecht",
+    "BE_OG_008": "Strafprozessrecht",
+    "BE_AK_001": "Allg. Staats- und Verwaltungsrecht",
+    "BE_SRK_001": "Steuerrecht",
+    "BE_VB_001": "Oeffentliches Bau-, Raumplanungs- und Umweltrecht",
+    "BE_VB_002": "Allg. Staats- und Verwaltungsrecht",
+    "BE_VB_003": "Allg. Staats- und Verwaltungsrecht",
+    "BE_VB_004": "Allg. Staats- und Verwaltungsrecht",
+    "BE_VB_005": "sonstiges Abgabenrecht",
+    "BE_VB_006": "Steuerrecht",
+    "BL_SG_001": "Steuerrecht",
+    "BL_ZMG_001": "Strafprozessrecht",
+    "FR_TC_004": "Familienrecht",
+    "FR_TC_005": "Strafprozessrecht",
+    "GE_CAPJ_001": "Allg. Staats- und Verwaltungsrecht",
+    "GE_CJ_003": "Arbeitsvertragsrecht",
+    "GE_CJ_004": "Miet- und Pachtrecht",
+    "GE_CJ_008": "Allg. Staats- und Verwaltungsrecht",
+    "GE_CJ_010": "Materielles Strafrecht",
+    "GE_TP_001": "Materielles Strafrecht",
+    "GE_TAPI_001": "Steuerrecht",
+    "GR_KG_003": "Schuldbetreibungs- und Konkursrecht",
+    "GR_KG_005": "Strafprozessrecht",
+    "GR_KG_010": "Allg. Staats- und Verwaltungsrecht",
+    "GR_KG_012": "Schuldbetreibungs- und Konkursrecht",
+    "JU_TC_003": "Materielles Strafrecht",
+    "LU_BKD_001": "Allg. Staats- und Verwaltungsrecht",
+    "NE_TC_009": "Materielles Strafrecht",
+    "NE_TC_011": "Materielles Strafrecht",
+    "NE_TC_014": "Materielles Sozialversicherungsrecht",
+    "NE_ARA_001": "Allg. Staats- und Verwaltungsrecht",
+    "NE_ASL_001": "Allg. Staats- und Verwaltungsrecht",
+    "NE_ASS_001": "Allg. Staats- und Verwaltungsrecht",
+    "NE_CA_001": "Allg. Staats- und Verwaltungsrecht",
+    "NE_CN_001": "Allg. Staats- und Verwaltungsrecht",
+    "SG_KG_003": "Allg. Staats- und Verwaltungsrecht",
+    "SG_ABSK_001": "Schuldbetreibungs- und Konkursrecht",
+    "SG_VB_001": "Oeffentliches Bau-, Raumplanungs- und Umweltrecht",
+    "SO_OG_001": "Strafprozessrecht",
+    "SO_OG_002": "Strafprozessrecht",
+    "SO_OG_005": "Schuldbetreibungs- und Konkursrecht",
+    "SO_OG_006": "Materielles Strafrecht",
+    "SO_STG_001": "Steuerrecht",
+    "SZ_KG_003": "Materielles Strafrecht",
+    "TI_CATI_001": "Steuerrecht",
+    "TI_TE_001": "Enteignungsrecht",
+    "TI_TRAP_001": "Materielles Strafrecht",
+    "TI_TRAP_002": "Strafprozessrecht",
+    "TI_TPC_001": "Materielles Strafrecht",
+    "TI_CARP_001": "Materielles Strafrecht",
+    "TI_GPC_001": "Strafprozessrecht",
+    "TI_CRP_001": "Strafprozessrecht",
+    "TI_GIAR_001": "Strafprozessrecht",
+    "VD_TC_003": "Materielles Strafrecht",
+    "VD_TC_005": "Allg. Staats- und Verwaltungsrecht",
+    "VS_AG_001": "Arbeitsvertragsrecht",
+    "VS_SRK_001": "Steuerrecht",
+    "ZG_VG_004": "Allg. Staats- und Verwaltungsrecht",
+    "ZH_BK_004": "Miet- und Pachtrecht",
+    "ZH_OG_003": "Strafprozessrecht",
+    "ZH_OG_005": "Allg. Staats- und Verwaltungsrecht",
+    "ZH_SRK_001": "Steuerrecht"
```

Comment:
How did you determine these? Are these only the ones where we have only one legal area?

Reply:
Also extracted from the `legal_info/court_chambers_extended.json` file and mapped using the `legal_info/law_areas.json`
Code for this can be found in the same place (`scrc/utils/chamber_areas_from_law_codes.py`)

Yes to the second question.